data/korath/korath.txt

```diff
@@ -211,10 +214,35 @@ fleet "Korath Ember Waste Raid"
 	outfitters "Korath Basics" "Korath Exiles"
 	personality
 		disables plunders opportunistic harvests
-	variant
+	variant 7
 		"Korath Raider (Ember)"
 		"Korath Chaser" 2
+	variant
+		"Korath Raider (Ember Shunt)"
+		"Korath Chaser" 2
 
+fleet "Korath Miners"
+	government "Korath"
+	names "korath"
+	fighters "korath"
+	cargo 0
+	personality
+		timid disables plunders frugal mining harvests
+	variant
+		"Korath Dredger (Digger)"
+		"Korath Chaser (Digger)" 4
+
+fleet "Korath Civilian Miners"
+	government "Korath (Civilian)"
+	names "korath"
+	fighters "korath"
+	cargo 0
+	personality
+		heroic disables frugal mining harvests coward forbearing
```

Comment:
Because these are civilian mining fleets, using the civilian government, which stay exclusively inside Korath space.  You'll see below they use different variants as well.